js/ad_manager_template.js

```diff
@@ -129,8 +129,9 @@ OO.Ads.manager(function(_, $) {
      * @method AdManager#cancelAd
      * @public
      * @param {object} ad The ad object to cancel
+     * @param {object} params An object containing information about the cancellation
```

Comment:
could i ask you to make the tiny change of adding something to the JSDoc like this:
"... about the cancellation.  It will include the following fields:"